src/rust/plugin-registry/tests/test_deploy_plugin.rs

```diff
@@ -135,6 +129,58 @@ async fn test_deploy_sysmon_generator() -> eyre::Result<()> {
     Ok(())
 }
 
+#[ignore] // until we have an Analyzer Execution Sidecar
```

Comment:
(I have another branch where I rebased atop analyzer-exectuion-sidecar and un-ignore this. I'm honestly down to just remove this test entirely instead of partially committing it here if that's preferable to people?)